berry_fix/charmap.txt

```diff
@@ -414,7 +414,7 @@ SIZE = FC 06 @ note that anything other than "SMALL" is invalid
 UNKNOWN_7 = FC 07
 PAUSE = FC 08 @ manually print the wait byte after this, havent mapped them
 PAUSE_UNTIL_PRESS = FC 09
-UNKNOWN_A = FC 0A
+WAIT_FOR_SE = FC 0A
```

Comment:
This is `WAIT_SE` in [pokefirered](/pret/pokefirered/blob/master/charmap.txt#L429)

Reply:
Resolved in 846e68b692655beeec18d6234eb5fbfeef1049d7